packages/image_picker/lib/image_picker.dart

```diff
@@ -44,7 +44,11 @@ class ImagePicker {
   static const MethodChannel _channel =
       MethodChannel('plugins.flutter.io/image_picker');
 
-  /// Returns a [File] object pointing to the image that was picked.
+  /// Returns a [File] object pointing to the image that was picked or throws
```

Comment:
nits: 
```
Returns a [File] object pointing to the image that was picked.
Throws ...
```
Also, the  message might be a little too definitive as there might be other platform exception thrown due to some error caused in the platform. (Like some iOS errors unknown to us), we might want to re-word it a little. Maybe something like "The method could throw a [PlatformException] if..."

Reply:
Hi @cyanglaz

Thanks for the feedback! I update the docs to explicit that the method could throw a `PlatformException` but left the description of the possible exceptions, because I think it is necessary for developers to be aware of the common exceptions that they might have to deal with when reading the documentation. If on the method docs is not the best place, could it be explicit on README.md?

Thanks,
freitzzz

Reply:
Sorry I didn't make myself clear enough. I meant what you have done is mostly ok, just that we need to indicate there might be other errors besides the ones that you have there.

Reply:
> Sorry I didn't make myself clear enough. I meant what you have done is mostly ok, just that we need to indicate there might be other errors besides the ones that you have there.

Got it! Just updated to reference that PlatformException could also be thrown due to an unknow error. Is it ok for now, or do you think it needs to be rephrased?

Reply:
@cyanglaz Let me know if you still want these changes

Reply:
absolutely! If you could rebase and resolve the conflicts, I will land this! Thanks again.